examples/dt/neutron/CRD_files/neutron_az_ovn.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+spec:
+  neutron:
+    template:
+      customServiceConfig: |
+        [DEFAULT]
+        dns_domain = openstackgate.local
+        vlan_transparent = true
+        network_scheduler_driver = neutron.scheduler.dhcp_agent_scheduler.AZAwareWeightScheduler
+        dhcp_load_type = networks
+        default_availability_zones = zone-1,zone-2
+        igmp_snooping_enable = true
+        global_physnet_mtu = 9000
+
+        [ovs]
+        igmp_snooping_enable = true
+    
+        [ovn]
+        ovn_emit_need_to_frag = true
+  
+        [ml2]
+        path_mtu = 9000
```

Comment:
considering this dt is targetting network nodes, may be we should disable ovn controller on control plane?

Reply:
I'm not sure where is the information that this DT is targetting deployment with networker nodes but if that's true then yes, we should definitely disable ovn-controller on Podified control plane as networker nodes should do that job :)

Reply:
Added a comment in DT to disable ovn-controller in controller nodes